cvmfs/cvmfs_config

```diff
@@ -1231,9 +1235,13 @@ add_user_to_group_fuse() {
 check_auto_mounter() {
   case $sys_arch in
     Linux )
-      $service autofs status > /dev/null 2>&1
-      return $? ;;
-
+        if [[ $service == *systemctl ]]; then
```

Comment:
Archlinux does not know any binary named `service`. In addition to that, the ordering of arguments accepted by systemctl seems to be different than the one used by `service`:
`service status autofs` vs. `systemctl autofs status`.